### PR TITLE
fix: allow zap comments

### PR DESCRIPTION
**Pull request by verbiricha**

### CODE REVIEW

File: src/Element/SendSats.tsx [hunk 1]
```diff
@@ -77,6 +77,8 @@ export default function LNURLTip(props: LNURLTipProps) {
   const { formatMessage } = useIntl();
   const publisher = useEventPublisher();
   const horizontalScroll = useHorizontalScroll();
+  const canComment =
+    (payService?.commentAllowed ?? 0) > 0 || payService?.nostrPubkey;
```
Comment (v0l) [inline src/Element/SendSats.tsx:81]: Having a `nostrPubkey` doesnt mean you can pass comment, only `commentAllowed > 0`

File: src/Util.ts [hunk 2]
```diff
@@ -108,24 +108,15 @@ export const Reaction = {
...
-      return Reaction.Positive;
-    case "🤙":
-      return Reaction.Positive;
-    case "❤️":
-      return Reaction.Positive;
-    case "👍":
-      return Reaction.Positive;
-    case "💯":
-      return Reaction.Positive;
     case "+":
```
Comment (v0l) [inline src/Util.ts:111]: Might as well also remove `+` and default to `Positive`

File: src/Element/SendSats.tsx (hunk 1 above)
Comment (verbiricha) [reply src/Element/SendSats.tsx:81]: zaps do allow comments tho, they are part of the zap not the invoice comment afaict

Comment (v0l) [reply src/Element/SendSats.tsx:81]: Yes but this is for the `?comment=` param, not the comment of the zap note, i guess it doesn't matter if you pass comment arg and its not supported for zaps endpoint

**Review by v0l — APPROVED** [review]

File: src/Element/SendSats.tsx (hunk 1 above)
Comment (verbiricha) [reply src/Element/SendSats.tsx:81]: will not send param if not supported by lnurl endpoint, good point

**Pull request merged**